tsdoc-config/src/TSDocConfigFile.ts

```diff
@@ -217,6 +226,23 @@ export class TSDocConfigFile {
     this._tagDefinitions.push(tagDefinition);
   }
 
+  /**
+   * Adds a new item to the `allowedHtmlTags` array.
+   */
+  public addAllowedHtmlTag(htmlTag: string): void {
+    this._specifiesAllowedHtmlTags = true;
+    this._allowedHtmlTags.add(htmlTag);
+  }
+
+  /**
+   * Removes the explicit list of allowed html tags. When no tags are explicitly listed,
+   * all tags will be permitted.
```

Comment:
In this representation, is it possible to allow an empty set (i.e. disallow all tags)?

Reply:
It is not, no. Is that a desired feature? If so, I could change it so that `null` allows all tags, and `[]` forbids all tags.